isogeny_primes.py

```diff
@@ -818,6 +924,52 @@ def get_pre_type_one_two_primes(K, norm_bound=50, loop_curves=False):
         q_dict_collapsed = gcd(list(q_dict.values()))
         tracking_dict_inv_collapsed[eps] = q_dict_collapsed
 
+    if use_PIL and h_K > 1:
```

Comment:
The `get_pre_type_one_two_primes` is getting a bit long and unwieldy. I think it is good to split it up into multiple functions. I think it makes sense to put some of the `use_PIL` stuff in a separate function that is called `get_PIL_integers` in the same spirit as the `get_C_integers`, and then have here only the code that mergers the results of the get_PIL_integers` with the other integers.

Reply:
Good point -- next commit will fix this.

Reply:
I've also taken the opportunity to split other stuff from this method into separate functions.